domains/moodle.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "nguyenkhanhhoa",
+        "email": "coolie.itman@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "m00dle.dynv6.net"
```

Comment:
Unable to access CNAME, please provide screenshot.

Reply:
> Unable to access CNAME, please provide screenshot.
My site's developing.
Please recheck again
![moodle](https://github.com/is-a-dev/register/assets/36908913/bbae5378-35e4-42ac-863d-59e7ff1e160a)